tests/__tests__/auto-cleanup-skip.js

```diff
@@ -0,0 +1,20 @@
+import '@testing-library/jest-dom/extend-expect'
```

Comment:
```suggestion
```

`extend-expect` is not in use in the test, so we can skip importing it

Reply:
ok. I will remove it.